src/wp-admin/css/customize-nav-menus.css

```diff
@@ -665,13 +665,17 @@
 #available-menu-items .menu-item-handle.item-added .item-title,
 #available-menu-items .menu-item-handle.item-added:hover .item-add,
 #available-menu-items .menu-item-handle.item-added .item-add:focus {
-	color: #8c8f94;
+	color: #1d2327;
 }
 
 #available-menu-items .menu-item-handle.item-added .item-add:before {
 	content: "\f147" / '';
 }
 
+#available-menu-items .menu-item-handle.item-added:hover .item-add:before {
```

Comment:
```suggestion
#available-menu-items .menu-item-handle.item-added:hover .item-add:before,
#available-menu-items .menu-item-handle.item-added .item-add:focus:before {
```

I am not convinced that the icon should change on `:hover`, but replacing the button icon on `:focus` could help more. This suggestion applies the plus icon to both `:hover` and `:focus` states.

Plus icon on `:focus` (with Gray 50 for text):
<img width="304" height="174" alt="Plus icon on focus" src="https://github.com/user-attachments/assets/9853c80b-7a13-435e-97a2-49f30dea9caf" />

Hover:
<img width="304" height="174" alt="Plus icon on hover" src="https://github.com/user-attachments/assets/a5e81f76-64c6-470c-886d-7529f7d3ebc9" />